tests/utils/test_check_shell_command.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from goose.utils.check_shell_command import is_dangerous_command
+
+
+@pytest.mark.parametrize(
+    "command",
+    [
+        "rm -rf /",
+        "git push origin master",
+        "sudo reboot",
+        "mv /etc/passwd /tmp/",
+        "chmod 777 /etc/passwd",
+        "chown root:root /etc/passwd",
+        "mkfs -t ext4 /dev/sda1",
+        "systemctl stop nginx",
+        "reboot",
+        "shutdown now",
+        "echo hello > ~/.bashrc",
+    ],
+)
+def test_dangerous_commands(command):
+    assert is_dangerous_command(command)
```

Comment:
nit: probably want to assert some of the inverse, with safe commands too

Reply:
that's what the second test does.